web/components/feed/feed-comments.tsx

```diff
@@ -231,31 +192,31 @@ export function FeedComment(props: {
                 />
               </>
             )}
-          <>
-            {bought} {money}
-            {contract.outcomeType !== 'FREE_RESPONSE' && betOutcome && (
-              <>
-                {' '}
-                of{' '}
-                <OutcomeLabel
-                  outcome={betOutcome ? betOutcome : ''}
-                  value={(matchedBet as any).value}
-                  contract={contract}
-                  truncate="short"
-                />
-              </>
-            )}
-          </>
+          {bought} {money}
+          {contract.outcomeType !== 'FREE_RESPONSE' && betOutcome && (
```

Comment:
Seems suspicious that this doesn't also exclude Multiple choice. I see that that's how it was though...